src/components/KonnectorSync.jsx

```diff
@@ -1,15 +1,26 @@
+import styles from '../styles/KonnectorSync'
+
 import React from 'react'
 import { translate } from 'cozy-ui/react/I18n'
 
 import DescriptionContent from './DescriptionContent'
 
-export const KonnectorSync = ({ t, f, date }) => {
+export const KonnectorSync = ({ t, f, date, error, editing, submitting, onForceConnection }) => {
   return (
     <div>
       { date && <DescriptionContent
         title={t('account.message.synced.title')}
         messages={[t('account.message.synced.last_sync', { date: f(date, t('account.message.synced.date_format')) })]}
       /> }
+      { !editing && !error && <div className={styles['account-forceConnection']}>
```

Comment:
I could be missing a point but I don't get why an error should cause the sync button to be hidden. What reason do you see ?